src/main/rx/spektrum.c

```diff
@@ -43,6 +43,7 @@
 #define SPEKTRUM_1024_CHANNEL_COUNT 7
 
 #define SPEK_FRAME_SIZE 16
+#define SPEKTRUM_TIME_NEEDED_PER_FRAME 5000
```

Comment:
Inter-frame is what i thought. Inter-byte confuses me. Individual bytes don't have this amount of spacing.
It is for packet synchronization, to know when a new packet starts.
It was a hard coded time, so i made a define for it.
Maybe better to call it SPEKTRUM_NEEDED_FRAME_INTERVAL